packages/marketplace/src/components/pages/settings/developer-settings-organisation-tab/developer-settings-organisation-tab.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react'
+import { FlexContainerResponsive, Content, FlexContainerBasic } from '@reapit/elements'
+import { Tabs } from '../tabs'
+
+const DevelperSettingsOrganisationTabPage: React.FC<{}> = () => {
```

Comment:
you can remove <{}> it if there is no props const DevelperSettingsBillingTabPage: React.FC = () => {